test/PriceOracle.test.ts

```diff
@@ -16,7 +16,8 @@ before(async function () {
   others = await getUnnamedAccounts();
 });
 
-const increaseRate = [0, 1036, 1027, 1025, 1024, 1023, 1021, 1017, 1012];
+const increaseRate = [1036, 1031, 1027, 1025, 1024, 1023, 1021, 1017, 1012];
+const roundLength = 152;
```

Comment:
this is already defined in the Redistribution test. would be nice to define it in utils instead (in one place).

Reply:
yes, that would make sense. just with new values and in other PR